package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "business-edit-ui",
-  "version": "3.9.15",
+  "version": "3.10.0",
```

Comment:
Adding a feature (filing type) so it's a good time to increase the minor version.